crates/sui/src/validator_commands.rs

```diff
@@ -454,6 +467,66 @@ impl SuiValidatorCommand {
                     serialized_data,
                 }
             }
+            SuiValidatorCommand::BridgeCommitteeRegistration {
+                bridge_node_config_path,
+                bridge_authority_url,
+            } => {
+                let bridge_config = BridgeNodeConfig::load(bridge_node_config_path)
+                    .expect("Couldn't load BridgeNodeConfig");
```

Comment:
Can we add more explanation to this error. Is the path wrong, is the path missing, etc? What should the user do when this error happens?